otter/test/test_cloud_client.py

```diff
@@ -405,3 +414,162 @@ def test_change_clb_node(self):
 
         # all the common failures
         self.assert_parses_common_clb_errors(expected.intent, eff)
+
+
+class NovaClientTests(SynchronousTestCase):
+    """
+    Tests for Nova client functions, such as :obj:`set_nova_metadata_item`.
+    """
+    def _setup_for_set_nova_metadata_item(self):
```

Comment:
I have definitely become slightly more predisposed to avoid `setUp` methods, because I used to write a lot of `setUp` code which put tons of attributes on `self` that I constantly had to look up every time I added a test.  So yes, this was sort of a desire to avoid them, but the main reason for the two different setup functions was that there are some common Nova errors (like rate limiting, no such server) that can share the same assertions.

Maybe a better way to do this would be to put the assertions in their own mixin?  And then go one test suite per client function, with it's own `setUp`?